jdaviz/app.py

```diff
@@ -1661,7 +1661,7 @@ def _on_new_viewer(self, msg, vid=None, name=None):
             viewer.state.x_att = x
 
         # Create the viewer item dictionary
-        if name is None:
+        if name is None and self.config != 'imviz':
             name = viewer.__class__.__name__
```

Comment:
Looks like this part was from before my time even though blame put it on #907 , there I was simply trying to preserve existing behavior. I don't see why we cannot set it the same as ID if it is not set.